src/components/battery/BatteryController.cpp

```diff
@@ -1,49 +1,92 @@
 #include "BatteryController.h"
 #include <hal/nrf_gpio.h>
+#include <nrfx_saadc.h>
 #include <libraries/log/nrf_log.h>
 #include <algorithm>
+#include <math.h>
 
 using namespace Pinetime::Controllers;
 
+#define BATTERY_VOL NRF_SAADC_INPUT_AIN7   // 31 Analog
```

Comment:
I think these constants and variables could be declared as member variables of the class `Battery`